### ISSUE TITLE
Source search could be better

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/CantusDB#1632]
If I search North Vancouver in Browse Sources, I get 29 results (although only one source is from North Vancouver)--because 2 sources mention Vancouver and another 27 mention "north". The search "North Vancouver" in quotes gets me nothing. If I search Vancouver I can get the source I want, but it seems counterintuitive to take out information to get what I need. 
Similarly, searching Tours Bibliotheque Municipale gives me 29 sources; adding in the number 149 brings it up to 83. For some reason in this case putting it in quotes still gives me 49 results, not 0 as it did for North Vancouver. Not sure why that is. 
Ideally at least one of "Tours Bibliotheque Municipale" in quotes or out of quotes should get me to the sources from that place!

**Comment by dchiller** [comment]
Yeah, this is buggy! Would be great to get to this in our next production update.

**Comment by dchiller** [comment]
Another aspect of this bugginess:
searching for "bibliotheque" returns sources that have "bibliothèque" in the source summary or bibliography, but searching for "bibliothèque" returns more sources (esp. sources that have "bibliothèque" in the institution name). We should fix this discrepancy while we're at it.
